### ISSUE TITLE
Use Youtube title for video metadata instead of the Content ID track title

### ISSUE THREAD

**Issue by anonamouslyginger** [tubearchivist/tubearchivist#241]
## tldr;
Note that this is a known issue on the `yt-dlp` repository. See [https://github.com/yt-dlp/yt-dlp/issues/3146](https://github.com/yt-dlp/yt-dlp/issues/3146).

The [suggested solution](https://github.com/yt-dlp/yt-dlp/issues/3146#issuecomment-1075666330) is to add `--parse-metadata "title:%(meta_title)s"` to the `yt-dlp` command.

## Issue Explanation

When downloading a YouTube video containing music which has been detected and linked by YouTube (see the description of the video) the downloaded metadata for that video contains the metadata of the music video instead of the archived video.

This issue is only visible when the "metadata embed" configuration option in the UI is set to True.
The UI and Elasticsearch data archived is correct for these videos however when importing these videos into another player (e.g. Plex), Plex displays the incorrect metadata which was embedded into the video.

## Issue Example

Archived/Downloaded YouTube Video: [Animation Supercut - Neebs Gaming](https://www.youtube.com/watch?v=NbBoN7J4n48)
Contains Music Video: [Call to Adventure - Comedy](https://www.youtube.com/watch?v=BX7PAL2kDO8)

As seen below, Movie name, Album and Performer are incorrect.

```
$ mediainfo "/raid-pool/yt/Neebs Gaming/20220325_NbBoN7J4n48_Animation Supercut - Neebs Gaming.mp4"
```
```
General
Complete name                            : /raid-pool/yt/Neebs Gaming/20220325_NbBoN7J4n48_Animation Supercut - Neebs Gaming.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/iso2/avc1/mp41)
File size                                : 292 MiB
Duration                                 : 57 min 45 s
Overall bit rate                         : 706 kb/s
Movie name                               : Call to Adventure - Comedy
Album                                    : Call to Adventure - Comedy
Performer                                : Kevin Macleod
Description                              : Neebs Gaming Animation Supercut.  / ► SUBSCRIBE -------  http://bit.ly/1NOKqlU /  / ►SUPPORT us on PATREON / https://www.patreon.com/neebsgaming /  / ► Neebs Gaming is powered by Xidax PCs, check them out here! /      http://mbsy.co/gFZJH /  / -----------Follow us on Social Media-------------------------- / ► https://www.Facebook.com/NeebsGaming / ► https://Twitter.com/Neebsofficial / ► http://www.neebsgaming.net / ------------------------------------------------------------------------------- / Looking for more videos from NEEBS?  Check out these playlist! / CARTOONS ----- https://bit.ly/2RASc99 / 7Days to Die ----- https://bit.ly/32Hb9gV / Scrap Mechanic ----- https://bit.ly/2RDhdQZ / Minecraft ----- https://bit.ly/35R51ol / GTA V ----- https://bit.ly/33CxwmP / Subnautica ----- https://bit.ly/2E9g0xJ
Recorded date                            : 20220325
Writing application                      : Lavf59.22.100
Comment                                  : https://www.youtube.com/watch?v=NbBoN7J4n48
LongDescription                          : Neebs Gaming Animation Supercut.  / ► SUBSCRIBE -------  http://bit.ly/1NOKqlU /  / ►SUPPORT us on PATREON / https://www.patreon.com/neebsgaming /  / ► Neebs Gaming is powered by Xidax PCs, check them out here! /      http://mbsy.co/gFZJH /  / -----------Follow us on Social Media-------------------------- / ► https://www.Facebook.com/NeebsGaming / ► https://Twitter.com/Neebsofficial / ► http://www.neebsgaming.net / ------------------------------------------------------------------------------- / Looking for more videos from NEEBS?  Check out these playlist! / CARTOONS ----- https://bit.ly/2RASc99 / 7Days to Die ----- https://bit.ly/32Hb9gV / Scrap Mechanic ----- https://bit.ly/2RDhdQZ / Minecraft ----- https://bit.ly/35R51ol / GTA V ----- https://bit.ly/33CxwmP / Subnautica ----- https://bit.ly/2E9g0xJ
...
```

## Desired Behavior

Given that I am downloading the target video which contains the music for only a small portion of the video, I would like the metadata to reflect the downloaded video. However, I can predict that some other users would like the opposite behavior. I would argue that as this tool is archiving the target video, the metadata should reflect that.

If a change to the default behavior is not acceptable, then a boolean configuration option, or a free form 'extra yt-dlp' configuration option would be fine.

If no change is desired at all, then a warning in the UI would be great.

**Comment by bbilly1** [comment]
Sorry, for some reason, your issue only just now showed up in the feed. No idea, did GitHub have a hiccup?

I think I understand the problem. How does that effect videos without music ids? Is that going to change anything for that?

**Comment by anonamouslyginger** [comment]
Ah, this issue was marked as “spammy” so each time I commented I was blocked. It should be visible again @bbilly1

**Comment by bbilly1** [comment]
Thank you for looking into this. Sounds good, please make a PR with your changes.

> resulting in 15 (unsure where 3 went) videos downloaded

Tube Archivist will skip videos if *yt-dlp* can't read it, from your list that would be:
- https://www.youtube.com/watch?v=DAqGcy4kmtQ
- https://www.youtube.com/watch?v=ThNrHfWKQvs
- https://www.youtube.com/watch?v=28JJ4FmVPkA

That's why you only see the 15 out of 18 in your queue.
